paper/preamble.tex

```diff
@@ -6,6 +6,48 @@
 \Copyright{Jan de Muijnck-Hughes, Guillaume Allais, and Edwin Brady}
 % TODO mandatory, please use full first names. LIPIcs license is "CC-BY";  http://creativecommons.org/licenses/by/3.0/
 
+%\begin{CCSXML}
+%<ccs2012>
+%   <concept>
+%       <concept_id>10003752.10003790.10011740</concept_id>
+%       <concept_desc>Theory of computation~Type theory</concept_desc>
+%       <concept_significance>500</concept_significance>
+%       </concept>
+%   <concept>
+%       <concept_id>10011007.10010940.10010992.10010998.10010999</concept_id>
+%       <concept_desc>Software and its engineering~Software verification</concept_desc>
+%       <concept_significance>500</concept_significance>
+%       </concept>
+%   <concept>
+%       <concept_id>10011007.10011006.10011008.10011009.10011012</concept_id>
+%       <concept_desc>Software and its engineering~Functional languages</concept_desc>
+%       <concept_significance>500</concept_significance>
+%       </concept>
+%   <concept>
+%       <concept_id>10011007.10011006.10011039</concept_id>
+%       <concept_desc>Software and its engineering~Formal language definitions</concept_desc>
+%       <concept_significance>500</concept_significance>
+%       </concept>
+%   <concept>
+%       <concept_id>10011007.10011006.10011050.10011017</concept_id>
+%       <concept_desc>Software and its engineering~Domain specific languages</concept_desc>
+%       <concept_significance>500</concept_significance>
+%       </concept>
+%   <concept>
+%       <concept_id>10002950.10003714.10003732.10003733</concept_id>
+%       <concept_desc>Mathematics of computing~Lambda calculus</concept_desc>
+%       <concept_significance>500</concept_significance>
+%       </concept>
+% </ccs2012>
+%\end{CCSXML}
+
+\ccsdesc[500]{Theory of computation~Type theory}
```

Comment:
I never know whether to include this. Because we're using TT as a tool rather than actually writing about TT.